src/components/Form/Field.js

```diff
@@ -111,6 +114,12 @@ const arrowDownStyle = css({
   top: lineHeight + fieldHeight / 2 - 3,
   cursor: 'pointer'
 })
+const iconWrapperStyle = css({
+  position: 'absolute',
+  right: 3,
+  top: fieldHeight - (fieldHeight * .75 / 2),
+  cursor: 'pointer'
```

Comment:
IMO rm, the icon element should decide if it is clickable and what cursor it should have, but can be overwritten either way